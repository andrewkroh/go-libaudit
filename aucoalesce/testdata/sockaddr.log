type=SYSCALL msg=audit(1492027062.361:1316151): arch=c000003e syscall=46 success=yes exit=10 a0=14 a1=7ffe911f2610 a2=4000 a3=ffffffffffffffa9 items=1 ppid=1 pid=385 auid=4294967295 uid=0 gid=0 euid=0 suid=0 fsuid=0 egid=0 sgid=0 fsgid=0 tty=(none) ses=4294967295 comm="systemd-logind" exe="/usr/lib/systemd/systemd-logind" subj=system_u:system_r:systemd_logind_t:s0 key=(null)
type=SOCKADDR msg=audit(1492027062.361:1316151): saddr=01002F72756E2F73797374656D642F6E6F74696679
type=CWD msg=audit(1492027062.361:1316151):  cwd="/"
type=PATH msg=audit(1492027062.361:1316151): item=0 name="/run/systemd/notify" inode=6504 dev=00:13 mode=0140777 ouid=0 ogid=0 rdev=00:00 obj=system_u:object_r:init_var_run_t:s0 objtype=NORMAL
