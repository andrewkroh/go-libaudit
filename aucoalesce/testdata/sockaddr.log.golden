{
  "@timestamp": "2017-04-12T19:57:42.361Z",
  "cwd": "/",
  "path": [
    {
      "dev": "00:13",
      "inode": "6504",
      "item": "0",
      "mode": "0140777",
      "name": "/run/systemd/notify",
      "obj_domain": "init_var_run_t",
      "obj_level": "s0",
      "obj_role": "object_r",
      "obj_user": "system_u",
      "objtype": "NORMAL",
      "ogid": "0",
      "ouid": "0",
      "rdev": "00:00"
    }
  ],
  "sequence": 1316151,
  "sockaddr": {
    "address_family": "unix",
    "path": "/run/systemd/notify"
  },
  "syscall": {
    "a0": "14",
    "a1": "7ffe911f2610",
    "a2": "4000",
    "a3": "ffffffffffffffa9",
    "arch": "x86_64",
    "auid": "unset",
    "comm": "systemd-logind",
    "egid": "0",
    "euid": "0",
    "exe": "/usr/lib/systemd/systemd-logind",
    "exit": "10",
    "fsgid": "0",
    "fsuid": "0",
    "gid": "0",
    "name": "sendmsg",
    "pid": "385",
    "ppid": "1",
    "ses": "4294967295",
    "sgid": "0",
    "subj_domain": "systemd_logind_t",
    "subj_level": "s0",
    "subj_role": "system_r",
    "subj_user": "system_u",
    "success": "yes",
    "suid": "0",
    "tty": "(none)",
    "uid": "0"
  }
}
